platform/admin/core/admin-core-appservice/src/test/java/org/codice/ddf/admin/application/service/command/StatusApplicationCommandTest.java

```diff
@@ -42,7 +42,7 @@
 
     private static final String TEST_FEATURE_NAME = "TestFeature";
 
-    private Logger logger = LoggerFactory.getLogger(StatusApplicationCommand.class);
+    private Logger LOGGER = LoggerFactory.getLogger(StatusApplicationCommand.class);
```

Comment:
We're starting to get out of scope here, but it doesn't look like this LOGGER is being used in the class